specification/mediaservices/resource-manager/Microsoft.Media/2015-10-01/media.json

```diff
@@ -85,13 +94,9 @@
             },
             "examples": { 

```

Comment:
This is causing error in https://travis-ci.org/Azure/azure-rest-api-specs/jobs/273496691. Please fix it